### PR TITLE
apache tinkerpop gremlin console

### PR DESCRIPTION
**Pull request by hanabi1224**

### CODE REVIEW

File: bucket/apache-tinkerpop-gremlin-console.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "version": "3.4.3",
+    "description": "A REPL environment for interactive development and analysis Gremlin applications",
```
Comment (Ash258) [inline bucket/apache-tinkerpop-gremlin-console.json]:
```suggestion
    "description": "A REPL environment for interactive development and analysis Gremlin applications.",
```

File: bucket/apache-tinkerpop-gremlin-console.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "homepage": "https://tinkerpop.apache.org/docs/3.2.5/reference/#gremlin-console",
+    "license": "Apache-2.0",
+    "url": "https://www.apache.org/dist/tinkerpop/3.4.3/apache-tinkerpop-gremlin-console-3.4.3-bin.zip",
+    "hash": "sha512:799ae9417be1e4e020438f7652287445c3b3d8bf37ebe1b729a0efe0806fda8aa05e825e016fd93f386b9c37c19e11ff2d6347a13539febcdb4743aa2d76b395",
+    "extract_dir": "apache-tinkerpop-gremlin-console-$version",
```
Comment (Ash258) [inline bucket/apache-tinkerpop-gremlin-console.json]: Unevalueated version

File: bucket/apache-tinkerpop-gremlin-console.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "bin": "bin/gremlin.bat",
```
Comment (Ash258) [inline bucket/apache-tinkerpop-gremlin-console.json]:
```suggestion
    "bin": "bin\\gremlin.bat",
```

File: bucket/apache-tinkerpop-gremlin-console.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
+    "version": "3.4.3",
+    "description": "A REPL environment for interactive development and analysis Gremlin applications",
+    "homepage": "https://tinkerpop.apache.org/docs/3.2.5/reference/#gremlin-console",
```
Comment (Ash258) [inline bucket/apache-tinkerpop-gremlin-console.json]: Why specific version url?

File: bucket/apache-tinkerpop-gremlin-console.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    "bin": "bin/gremlin.bat",
+    "checkver": {
+        "url": "https://www.apache.org/dist/tinkerpop/",
+        "regex": "href=\"([\\d.]+)/"
+    },
+    "autoupdate": {
+        "url": "https://www.apache.org/dist/tinkerpop/$version/apache-tinkerpop-gremlin-console-$version-bin.zip",
+        "hash": {
+            "url": "$url.sha512"
+        }
```
Comment (Ash258) [inline bucket/apache-tinkerpop-gremlin-console.json]:
```suggestion
        },
        "extrect_dir": "apache-tinkerpop-gremlin-console-$version"
```

**Comment by hanabi1224** [comment]
Kindly ping

File: bucket/apache-tinkerpop-gremlin-console.json [hunk 6]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "3.4.3",
+    "description": "A REPL environment for interactive development and analysis of Gremlin applications.",
+    "homepage": "https://tinkerpop.apache.org/",
```
Comment (Ash258) [inline bucket/apache-tinkerpop-gremlin-console.json]:
```suggestion
    "homepage": "https://tinkerpop.apache.org",
```

**Pull request merged**